src/main/java/core/org/akaza/openclinica/service/StudyParticipantServiceImpl.java

```diff
@@ -500,10 +500,17 @@ public void startCaseBookPDFJob(JobDetail jobDetail,
 			    }//for-loop-1		   
 			    
 				mergedPdfFile = pdfService.mergePDF(pdfFiles, fullFinalFilePathName);
-				String footerMsg = "OpenClinica CaseBook ";
-				pdfService.addFooter(fullFinalFilePathName, footerMsg);
 				mergedPdfFileNm = mergedPdfFile.getName();
 				userService.persistJobCompleted(jobDetail, mergedPdfFileNm);
+				
+				// when add footer, need this file is ready -- can be read
+				while(!(mergedPdfFile.canRead())) {
+					;//wait
```

Comment:
instead of adding footer at the end, add it as you are merging the files in mergePDF method.

Reply:
done